po/pt.po

```diff
@@ -0,0 +1,914 @@
+msgstr "_Alcunha:"
+
+#: src/text.vala:334
+msgid "Offline"
+msgstr "Desligado"
+
+#: src/text.vala:335
+msgid "Online"
+msgstr "Ligado"
+
+#: src/text.vala:336
+msgid "Unreachable"
+msgstr "Inacessível"
+
+#: src/text.vala:337
+msgid "Protocol mismatch"
+msgstr "Protocolo incompatível"
+
+#: src/text.vala:338
+msgid "Conflicting address"
+msgstr "Conflito de endereço"
+
+#: src/text.vala:339
+msgid "C_onnect"
+msgstr "_Conectar"
+
+#: src/text.vala:340
+msgid "_Disconnect"
+msgstr "_Desconectar"
+
+#: src/text.vala:341
+msgid "Reconnect"
+msgstr "Reconectar"
+
+#: src/text.vala:342
+msgid "Connected"
+msgstr "Conectado"
+
+#: src/text.vala:343
+msgid "Disconnected"
+msgstr "Desconectado"
+
+#: src/text.vala:344
+msgid "Initializing…"
+msgstr "A inicializar..."
+
+#: src/text.vala:345
+msgid "Connecting…"
+msgstr "A conectar..."
+
+#: src/text.vala:346
+msgid "Logging in…"
+msgstr "A entrar..."
+
+#: src/text.vala:347
+msgid "Updating…"
+msgstr "A atualizar..."
+
+#: src/text.vala:349
+msgid "Hamachi is not configured"
+msgstr "O Hamachi não está configurado"
+
+#: src/text.vala:350
+msgid "You need to configure Hamachi before you can connect."
+msgstr "Tem que configurar o Hamachi antes de poder conectar."
+
+#: src/text.vala:351
+msgid "C_onfigure"
+msgstr "_Configurar"
+
+#: src/text.vala:353
+msgid "Hamachi is not installed"
+msgstr "O Hamachi não está instalado"
+
+#: src/text.vala:354
+msgid "Please download Hamachi and follow the installation instructions."
+msgstr "Por favor descarregue o Hamachi e siga as instruções de instalação."
+
+#: src/text.vala:355
+msgid "Hamachi version {0} is obsolete"
+msgstr "A versão do Hamachi {0} é obsoleta"
+
+#: src/text.vala:356
+msgid "Please download and install the latest Hamachi version."
+msgstr "Por favor descarregue e instale a versão mais recente do Hamachi."
+
+#: src/text.vala:357
+msgid "_Download"
+msgstr "_Descarregar"
+
+#: src/text.vala:359
+msgid "You are connected!"
+msgstr "Está conectado!"
+
+#: src/text.vala:360
+msgid "Click on the + button to add a network."
+msgstr "Clique no botão + para adicionar uma rede."
+
+#: src/text.vala:362
+msgid "Members:"
+msgstr "Membros:"
+
+#: src/text.vala:363
+msgid "{0} online, {1} total"
+msgstr "{0} ligados, {1} total"
+
+#: src/text.vala:364
+msgid "Owner:"
+msgstr "Proprietário:"
+
+#: src/text.vala:365
+msgid "You"
+msgstr "Você"
+
+#: src/text.vala:366
+msgid "Unknown"
+msgstr "Desconhecido"
+
+#: src/text.vala:367
+msgid "Unavailable"
+msgstr "Indisponível"
+
+#: src/text.vala:368
+msgid "Awaiting approval"
+msgstr "A aguardar aprovação"
+
+#: src/text.vala:369
+msgid "Locked:"
+msgstr "Trancado:"
+
+#: src/text.vala:370
+msgid "Approval:"
+msgstr "Aprovação:"
+
+#: src/text.vala:371
+msgid "Manually"
+msgstr "Manualmente"
+
+#: src/text.vala:372
+msgid "Automatically"
+msgstr "Automaticamente"
+
+#: src/text.vala:373
+msgid "Capacity:"
+msgstr "Capacidade:"
+
+#: src/text.vala:375
+msgid "_Go Online"
+msgstr "_FIcar Ligado"
```

Comment:
Letter `I` should be lowecase I presume, i.e. `Ficar` instead of `FIcar`.

Reply:
Done